www/js/app.js

```diff
@@ -64,34 +65,100 @@ define(['jquery', 'zimArchiveLoader', 'uiUtil', 'settingsStore','abstractFilesys
      */
     var selectedArchive = null;
     
-    // Set parameters and associated UI elements from the Settings Store
-    // DEV: The params global object is declared in init.js so that it is available to modules
-    params['storeType'] = settingsStore.getBestAvailableStorageAPI(); // A parameter to determine the Settings Store API in use
+    /**
+     * Set parameters from the Settings Store, together with any defaults
+     * Note that the params global object is declared in init.js so that it is available to modules
+     * WARNING: Only change these paramaeters if you know what you are doing
+     */
+    // The current version number of this app
+    params['appVersion'] = '3.2.3'; // **IMPORTANT** Ensure this is the same as the version number in service-worker.js
```

Comment:
Shouldn't we use the same "3.3-WIP" string as elsewhere?
I suppose you used these version numbers for testing purpose?

Reply:
Yes, it was for testing. Will revert.